src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -595,6 +602,8 @@ void LottieBuilder::updateStar(LottiePolyStar* star, float frameNo, Matrix* tran
         angle += dTheta * direction;
         longSegment = !longSegment;
     }
+    //ensure proper shape closure - may be important for modifiers
```

Comment:
This comment is just ambuguous, not helpful.  "may be important for modifiers"

Please make it sure or remove them.